tv-app/effecticeTV/app/build.gradle

```diff
@@ -24,6 +24,7 @@ android {
         }
 
         buildConfigField ("String", "apiLeaderUrl", localProperties['apiLeaderUrl'])
+        buildConfigField ("int", "slideShowPeriod", localProperties['slideShowPeriod'])
```

Comment:
There is not necessarily provide constants by `buildConfigFIeld`
The better way to create constants is singleton object like this:

```kotlin

object SliderProperties {
    val slideShowPeriod: Int = 10 // for example
}